users/clients.go

```diff
@@ -8,48 +8,56 @@ import (
 
 	"github.com/absmach/magistrala"
 	"github.com/absmach/magistrala/pkg/authn"
-	"github.com/absmach/magistrala/pkg/clients"
 )
 
 // Service specifies an API that must be fullfiled by the domain service
 // implementation, and all of its decorators (e.g. logging & metrics).
 //
 //go:generate mockery --name Service --output=./mocks --filename service.go --quiet --note "Copyright (c) Abstract Machines"
 type Service interface {
```

Comment:
Remove `User` prefix where not necessary, it's redundant due to package name `users.Service.Register` is sufficient, no need for `users.Service.RegisterUser`. Sam for `List`, `viewByUsername`, `updateRole` and so on. We will update this for Things and other services as well.

Reply:
Similar to Repository approach.